### ISSUE TITLE
Why MultiWOZ goal generator deletes some slots?

### ISSUE THREAD

**Issue by ghost** [thu-coai/ConvLab-2#191]
Why the goal generator of MultiWOZ deletes some slots?

https://github.com/thu-coai/ConvLab-2/blob/0451cb4c6a4f0d208752ec7f52788a2f62d6cf0d/convlab2/task/multiwoz/goal_generator.py#L170-L175

**Comment by zqwerty** [comment]
these slots are not used in the goal of multiwoz data. Actually, these two domains are quite simple and rare, and not included in multiwoz test data. So we excluded them for user simulator

**Comment by ghost** [comment]
But after deletion there are still "address" and "phone" in `self.ind_slot_dist['police']['reqt']`, and "phone" in `self.ind_slot_dist['hospital']['reqt']`.

Why just delete "postcode" in both two domains and "address" in "hospital"?
Why don't also delete address in "police"?
And why "phone" still exists in both two domains?

Thanks in advance for answering these questions.

**Comment by zqwerty** [comment]
I've found the reason. In the beginning, there is no `postcode` for `policy` and no `address` and `postcode` for `hospital` in MultiWOZ dataset. So we del these slots.
In [this commit](https://github.com/thu-coai/ConvLab-2/commit/b82732eae951b3dc957136f40b992a1904c9cbe5#diff-d70a6af7f354234f737e12875fabbd245e3ce84eac491f43e9665f2b375318c7), we found the unique values for these slots. However, we did not add these slots back in goal generator. Thank you for pointing out! We will fix it.
